apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -141,6 +141,23 @@ public static Long parseLong(String longText) {
     }
   }
 
+  /**
+   * @param shortText the number string to parse
+   * @return the specified text parsed into an {@link Long}
+   */
+  public static short parseShort(String shortText) {
+    /*
+     * Might seem silly to pull this out, but it makes the code a bit easier
+     * to read, and ensures that this parsing is standardized.
+     */
```

Comment:
I suggest getting rid of this low-value comment here (I know that it got copy pasted from the integer one)

Reply:
ok, if I remove this, I think we should remove them all; there's not a whole lot of value to the comment, and there's nothing wrong with code that handles particular inputs.